src/darwin/agent/launchd.js

```diff
@@ -252,3 +271,44 @@ function findClosestMachHeader(address) {
     cur = cur.sub(4096);
   }
 }
+
+function findInserterResume() {
+  if (_cachedInseterImpl !== null) {
+    if (_cachedInseterImpl.isNull()) {
+      return null;
+    }
+    return _cachedInseterImpl;
+  }
```

Comment:
I guess there's only a single caller so there's no need to optimize this?

Reply:
right!